spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+RSpec.describe PostsController, type: :controller do
+  describe 'GET #index' do
```

Comment:
- [ ] I can see you did great implementations in test. However,  your tests are failing. Kindly look through them. The steps that I'm doing to test this project.

1. clone this project
2. run bundle
3. run rails db:create, db:migrate 
4. rspec spec/ also tried bundle exec rspec spec

the result:
<img width="780" alt="Screenshot 2023-02-15 at 23 45 51" src="https://user-images.githubusercontent.com/69011963/219124304-ed34e583-4a20-4a1c-9270-c267e2b185f7.png">

Kindly make sure all testy are passing as expected.